Spigot-API-Patches/0300-Add-Mob-lookAt-API.patch

```diff
@@ -0,0 +1,82 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <blake.galbreath@gmail.com>
+Date: Fri, 14 May 2021 13:42:06 -0500
+Subject: [PATCH] Add Mob#lookAt API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Mob.java b/src/main/java/org/bukkit/entity/Mob.java
+index d726453c041a980576312b6bee96a07837f37974..f4b21cf9c1d7a09db9df81e07209add7946831ad 100644
+--- a/src/main/java/org/bukkit/entity/Mob.java
++++ b/src/main/java/org/bukkit/entity/Mob.java
+@@ -1,6 +1,8 @@
+ package org.bukkit.entity;
+ 
++import org.bukkit.Location;
+ import org.bukkit.loot.Lootable;
++import org.bukkit.util.Vector;
```

Comment:
Is this import used anywhere?

Also, I was under the impression that fully qualified names were still preferred, but idrk anymore.

Reply:
Ah, no. I _was_ going to add methods with Vector as a param, but decided against it since those are mainly used for directions and not positions. forgot to remove that import, though ^_^